### DISCUSSION TITLE
Timer for commands

### DISCUSSION THREAD

**Discussion by gallois** [wezterm/wezterm#5526] [General]
iTerm has a feature that's supported through scripting which allows you to display for how a command is currently running[1]. While not absolutely necessary, it can be pretty useful on some situations. I couldn't find anything in the API for that. Is it something possible currently?

[1] https://gitlab.com/gnachman/iterm2/-/issues/9572

**Comment by wez** [comment]
You can probably do something similar using user-vars and the user-var-changed event.
Take a look at https://wezfurlong.org/wezterm/recipes/passing-data.html

**Comment by gallois** [comment]
Makes sense! I thought user vars would be the way to go but wanted to check if I wasn't missing something. Thanks for the quick response!
